.gitignore

```diff
@@ -16,3 +16,5 @@
 .DS_Store
 .idea
 .vscode
+
+rust-toolchain.toml
```

Comment:
Can we add `rust-toolchain.toml` to the project? It would be nice to have the file with nightly rust.

Reply:
I think proper toolchain is individual for different systems